src/BuiltInTools/dotnet-watch/Internal/DotnetMuxer.cs

```diff
@@ -1,19 +0,0 @@
-﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System.Diagnostics;
-using System.IO;
-
-namespace Microsoft.DotNet.Watcher.Tools
-{
-    internal static class DotnetMuxer
-    {
-        static DotnetMuxer()
-        {
-            MuxerPath = Process.GetCurrentProcess().MainModule.FileName;
-            Debug.Assert(Path.GetFileNameWithoutExtension(MuxerPath) == "dotnet", $"Invalid muxer path {MuxerPath}");
```

Comment:
While I do like making things simpler, I feel we are missing out on this Debug.Assert with these changes. It makes anyone reading this code understand that we expect the current executable to be `dotnet`, and not some other host.

What do you think about keeping this class around and just changing:

```
MuxerPath = Environment.ProcessPath;
```

?

Reply:
I've brought back DotnetMuxer.